gis4wrf/core/transforms/wrf_netcdf_to_gdal.py

```diff
@@ -252,6 +253,8 @@ def set_band_metadata(fmt: GDALFormat) -> None:
                 data = var[band_idx - 1]
                 if is_4d:
                     data = data[extra_dim_index]
+                if ma.is_masked(data):
```

Comment:
@letmaik I checked and the data type is `<class 'numpy.ma.core.MaskedArray'>` however here `ma.is_masked(data)` returns `false` hence directly executing `band.WriteArray(data.astype(np_dtype, copy=False))` without unpacking the array.